app/api/views/opportunities.py

```diff
@@ -0,0 +1,73 @@
+from app.api import api
+from flask import request, jsonify
+from app.api.services import briefs
+
+
+@api.route('/opportunities', methods=['GET'])
```

Comment:
should this be `/brief` rather than `/opportunities` ?

Reply:
Perhaps, but I'd argue the data coming back here is specifically tailored to the opportunities page and would be unlikely to satisfy other pages that may want brief data, without them needing to augment the response with more data (which I don't think is a good idea as the opportunities page will start losing performance the more that's added to this response).